src/main/java/core/basesyntax/ColorSuppier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSuppier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int colorIndex = random.nextInt(Colors.values().length - 1);
+        return Colors.values()[colorIndex].name();
```

Comment:
```suggestion
        int colorIndex = random.nextInt(Colors.values().length);
        return Colors.values()[colorIndex].name();
```
You will never get the last one with '-1'

Reply:
Not fixed

Reply:
not fixed